snippets/markdown.json

```diff
@@ -1,4 +1,27 @@
 {
+    "YAML Header": {
+        "prefix": "---",
+        "body": [
+            "---",
+            "title: ${1:title}",
+            "author: ${2: author_name}",
+            "date: ${3:\"`r Sys.Date()`\"}",
+            "output: ${5:pdf_document}",
+            "---"
+        ]
+    },
+    "Insert Inline Chunck": {
+        "prefix": "`",
+        "body": "`${0}`"
+    },
+    "Insert Code Language": {
+        "prefix": "```",
+        "body": ["```${1:lang}", "${0}", "```"]
+    },
+    "Insert Block Equation": {
+        "prefix": "$$",
+        "body": ["\\$$", "${0}", "\\$$"]
+    },
```

Comment:
Could this be omitted in favor of users extending the snippets they load when interacting with markdown files? 

(I'm trying to understand your reasoning for the inclusion of this snippet)

Reply:
This is a copy from `rmarkdown.json`. I would suggest: either we introduce this here, or we delete the one in `rmarkdown.json`...